.changes/v2.17.0/503-enhancement.md

```diff
@@ -0,0 +1,3 @@
+* Added new field Hypervisor in QueryResultVMRecordType struct:
+    - String field containing the hostName value from the XML VMAdminRecord body retrieved
+    - This info can helps to link VM with a cluster since hypervisor is knowned
```

Comment:
```suggestion
* Added new field `Hypervisor` in `QueryResultVMRecordType` struct:
    * String field containing the hostName value from the XML VMAdminRecord body retrieved
    * This info can help to link VM with a cluster since hypervisor is known [GH-503]
```